microcosm_elasticsearch/indexing.py

```diff
@@ -0,0 +1,29 @@
+"""
+Index management.
```

Comment:
I would argue the most basic, useful things you could add, which we have in gaston today are:

- a cli for iterating over documents which match some query. We have a cli for iterating over all documents in: https://github.com/globality-corp/gaston/blob/develop/setup.py#L42 and optionally exporting documents to a file
- a cli for loading documents into a cluster from a file, we have one here: https://github.com/globality-corp/gaston/blob/develop/setup.py#L38
- a reindex cli for copying data from one index on a cluster to a new index. I think we have something like that here: https://github.com/globality-corp/gaston/blob/develop/setup.py#L45 but i can't remember the exact semantics of what we implemented